tests/components/soma/test_config_flow.py

```diff
@@ -0,0 +1,40 @@
+"""Tests for the Soma config flow."""
+from unittest.mock import patch
+
+from api.soma_api import SomaApi
+
+from homeassistant import data_entry_flow
+from homeassistant.components.soma import config_flow, DOMAIN
+from tests.common import MockConfigEntry
+
+
+MOCK_HOST = "123.45.67.89"
+MOCK_PORT = 3000
+
+
+async def test_form(hass):
+    """Test user form showing."""
+    flow = config_flow.SomaFlowHandler()
+    flow.hass = hass
+    result = await flow.async_step_user()
+    assert result["type"] == data_entry_flow.RESULT_TYPE_FORM
+
+
+async def test_import(hass):
```

Comment:
Here we're testing the import step abort functionality. We should also test the import step leading to creating an entry.

Reply:
I think that was the only missing flow path so this should be done now.